lms/djangoapps/projects/tests/test_projects.py

```diff
@@ -108,6 +114,32 @@ def do_delete(self, uri):
         response = self.client.delete(uri, headers=headers)
         return response
 
+    def test_projects_list_get(self):
+        """ Tests simple GET request - should return all projects """
+        response = self.do_get(self.test_projects_uri)
+        projects = response.data
+        self.assertEqual(len(projects), 2)
+        project1, project2 = projects[0], projects[1]
+        self.assertEqual(project1['id'], self.test_project.id)
+        self.assertEqual(project1['course_id'], self.test_project.course_id)
+        self.assertEqual(project1['content_id'], self.test_project.content_id)
+
+        self.assertEqual(project2['id'], self.test_project2.id)
+        self.assertEqual(project2['course_id'], self.test_project2.course_id)
+        self.assertEqual(project2['content_id'], self.test_project2.content_id)
+
+    def test_projects_list_get_filter_by_content_id(self):
+        """ Tests GET request with specified content_id - should return single project with matching content_id """
```

Comment:
It's possible to return more than one project record if we only search by content_id, so I think we need to additionally filter by course_id to narrow the set down further.